frontend/src/pages/breadcrumbs.tsx

```diff
@@ -0,0 +1,40 @@
+import BreadcrumbGroup from '@cloudscape-design/components/breadcrumb-group'
+import {BreadcrumbGroupProps} from '@cloudscape-design/components/breadcrumb-group/interfaces'
+import {CancelableEventHandler} from '@cloudscape-design/components/internal/events'
+
+type Page = 'clusters' | 'custom-images' | 'official-images' | 'users'
+
+const pageBreadcrumbItems: Record<Page, BreadcrumbGroupProps.Item> = {
+  clusters: {text: 'Clusters', href: '#clusters'},
```

Comment:
issue: as suggested in the other comment, please add i18n to the mix

Reply:
added in the next rev (coming up)